agent/client/client.go

```diff
@@ -378,7 +378,6 @@ LOOP:
 			case *agentv1.StartActionRequest:
 				responsePayload = &agentv1.StartActionResponse{}
 				if err := c.handleStartActionRequest(p); err != nil {
-					responsePayload = nil
```

Comment:
An empty payload caused an 'Internal Server Error' regardless of the error type returned.